src/tools/chip-cert/Cmd_ValidateCert.cpp

```diff
@@ -108,6 +108,11 @@ bool HandleOption(const char * progName, OptionSet * optSet, int id, const char
     {
     case 'c':
     case 't':
+        if (gNumCertFileNames >= kMaxCerts - 1)
```

Comment:
Is this off by one?  We are comparing number to number, not number to index.  Why -1?

Reply:
So the original fix was correct because the array size was improperly defined as `kMaxCerts - 1` - I fixed that in both places.